reactor-core/src/main/java/reactor/core/publisher/SignalLogger.java

```diff
@@ -191,8 +200,21 @@ else if (level == Level.SEVERE) {
 		}
 		return null;
 	}
+@Override
+	public Consumer<? super Context> onContextPropagateCall() {
+		if ((options & CONTEXT_PROPAGATE) == CONTEXT_PROPAGATE && (level != Level.INFO || log.isInfoEnabled())) {
+			return c -> log(SignalType.ON_CONTEXT, c, source);
+		}
+		return null;
+	}
 
-	String subscriptionAsString(@Nullable Subscription s) {
+	@Override
+	public Consumer<? super Context> onContextParentCall() {
+		if ((options & CONTEXT_PARENT) == CONTEXT_PARENT && (level != Level.INFO || log.isInfoEnabled())) {
+			return c -> log(SignalType.CURRENT_CONTEXT, c, source);
+		}
+		return null;
+	}	String subscriptionAsString(@Nullable Subscription s) {
```

Comment:
missing newline x2 (note a parallel PR #659 changes `subscriptionAsString` to static)